package.json

```diff
@@ -4,7 +4,7 @@
   "description": "transparently require() jsx from node",
   "main": "index.js",
   "dependencies": {
-    "react-tools": "^0.13.2"
+    "jstransform": ">=11.0.0"
```

Comment:
I did this for consistency with [jsx-loader](https://github.com/petehunt/jsx-loader/blob/8c9e7115a00ab3a449ea9c50130194f0a3e2ddfe/package.json#L7).

If jstransform is following semver, we could make this "11" and let `npm` worry about getting the latest patch version.  We could also make this a `peerDependency` and let the application developer choose which version of `jstransform` to use.

One frustrating thing about life before `>=` is that new visitors were gated on @petehunt taking the time to version bump.  It'd be nice if there was an easy way to make `jsx-loader` and `node-jsx` use a newer version of `jstransform`.

Should I change this to `"11"` and trust jstransform to bump the minor version for new visitors?